frontend/src/components/chat/ChatForm.js

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect, useRef } from "react";
+
+import { PaperAirplaneIcon } from "@heroicons/react/solid";
+import { EmojiHappyIcon } from "@heroicons/react/outline";
+import Picker from "emoji-picker-react";
+
+export default function ChatForm(props) {
+  const [message, setMessage] = useState("");
+  const [showEmojiPicker, setShowEmojiPicker] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isShowEmojiPicker, setIsShowEmojiPicker] = useState(false);
```